website/src/components/Survey/TrackedTextarea.tsx

```diff
@@ -11,10 +30,61 @@ interface TrackedTextboxProps {
   onTextChange: (event: React.ChangeEvent<HTMLTextAreaElement>) => void;
 }
 
+const killEvent = (e) => e.stopPropagation();
+
 export const TrackedTextarea = (props: TrackedTextboxProps) => {
+  const [wordLimitForLangDetection, setWordLimitForLangDetection] = React.useState(10);
   const backgroundColor = useColorModeValue("gray.100", "gray.900");
-
+  const [cookies, setCookie] = useCookies(["NEXT_LOCALE", "IGNORE_LANGUAGE_DETECTION"]);
   const wordCount = (props.text.match(/\w+/g) || []).length;
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const currentLanguage = cookies["NEXT_LOCALE"];
+  const ignoreLanguageDetection = cookies["IGNORE_LANGUAGE_DETECTION"] === "true";
+
+  const setIgnoreLanguageDetection = () => {
+    setCookie("IGNORE_LANGUAGE_DETECTION", "true", { path: "/" });
```

Comment:
Because it could be annoying for a multi-lingual person? I'm open to removing this feature. 

Concerning the suggestions by @andreaskoepf in Discord, the language check should also be more directed to the language of the preceding thread, currently it is checked against the user language. This requires some refactoring that might need some more input from your side (currently tasks are very generic so I don't have a way to access contents/content language).

Reply:
Given the prevelance of the problem I think we should remove this feature for now and add it later.  Wrong language submissions seem to be a major problem right now and we shouldn't make that easier.

I think the easier way to avoid these problems overall is to do all this language check stuff on the backend after submission and just not give a user points if they respond in the wrong language after tossing out the response.  I did that for Translate and it made things *so much* more reliable.

Reply:
I changed the code now so that live language detection cannot be disabled, nevertheless check intervals become longer as long as the same react state is preserved. See this as a quick check for keeping users off bad submissions early on.
I absolutely see the point of having later checks in the backend, maybe this should be done in another PR.